### ISSUE TITLE
Okay other jokers, here are some serious discussion points

### ISSUE THREAD

**Issue by steakknife** [responsible-financial-innovation-act22/RFIA-bill#138]
(I'm in tech from SV originally.) In 2010, I knew crypto (i.e., BTC) _**as "investments"**_ were high beta (risky) speculative instruments of suspicious stability, infosec integrity, and providing little trust in long-term survival or liquidity at exchange points. Mtgox later proved the latter suspicion correct.

As privacy transactional markets (amoral - for good, neutral, or criminality) to side-step ACH, SWIFT, Fedwire, CHIPS, RTGS, SCT/Inst, CIPS, etc. and credit card systems' commercial monopoly strangleholds and apply competitive pressures to traditional markets and instruments, they succeeded. Crypto coins' underlying value is often too unstable to depend on longer than a conversion for a goods purchase by a consumer and net settlement with a merchant. And not all stablecoins are perfectly liquid as cash equivalents either.

Two philosophies I don't agree with are:

- Paper currency demonetization where a person will be improperly and/or unnecessarily surveilled (hacking, corporate data mining, and/or government reporting) and/or cut off at  undemocratic whims of corporations or jurisdictions (domestic or foreign).

- Outlawing or completely unregulated crypto or NFTs are terrible ideas.

I think crypto's moving pieces and stakeholders including coin specifications and software maintainers, mixers, markets, wallets, and wallet storage devices (apps/USB) above a certain equivalent USD market cap or volume need lightweight fintech regulation to protect investors, users, and neighbor markets to deter gross malice, incompetence, and negligence. Such data elements might be regularly reporting to regulators their named principals/executives, information security strategies, investors, balance sheet, energy consumption trends for proof-of-work/-stake (PoW/PoS). Whether it's possible to capture clients' nationality or identity in a KYC "account" requirements if no account exists (a wallet) in a technical sense presents a policy decision hurdle.

tl;dr - Don't stomp out innovation for uses like:
- literal pennies are being exchanged to beta test a new technology platform
- a citizen of a repressive regime wishes to migrate and seek asylum without carrying as much in valuables or hard currency
- poor people accumulating value to escape bad situations
- a husband wants to buy their wife an "embarrassing" item without it appearing on a credit card or being associated with either of their e-commerce accounts or data-trafficked consumer histories
- a farmer wants to buy a tractor at auction without being subjected to Civil Asset Forfeiture

but don't facilitate:
- direct fraud
- criminal enterprises
- hostile state or nonstate activities
- inappropriate tax minimization

**Comment by CrazedCoding** [comment]
Ya know... Im starting to get the feeling law is just the religion of politics.

Seriously: how does anyone get paid to sit down and write so much intricately obfuscated trash?

I bet we could actually write a new GitHub repo implementing a new cryptocurrency in fewer characters than this ridiculous excuse for something that's somehow supposed to make our government less dysfunctional.

Just sayin'...

**Comment by CrazedCoding** [comment]
Elaboration: https://github.com/responsible-financial-innovation-act22/RFIA-bill/issues/104#issue-1283651093

**Comment by joelsgp** [comment]
> Im starting to get the feeling law is just the religion of politics.

so true milady! my understanding of politics is even less than my mostly imaginary understanding of technology, so I like to compare it to religion because the other ppl on reddit said god is dead. honestly though I would say law is the dark souls of merkle trees

**Comment by zathras** [comment]
This issue violates community standards.  There isn't a single mention of floppa, and it seems to take seriously the idea that cryptocurrency is somehow valuable.

I do realize I'm going out on a limb, making fun of someone "in tech from SV originally."  After all, there is no finer preparation for creating a new monetary and regulatory system than writing Python scripts and doing QA for websites.

Speaking seriously, this is one of the problems with the whole crypto scam/fad.  People think that anyone who knows some basics of coding somehow are somehow qualified to "disrupt" industries they know nothing about.  Humility is all too rare a quality in Silicon Valley.

**Comment by CrazedCoding** [comment]
> > Im starting to get the feeling law is just the religion of politics.
> 
> so true milady! my understanding of politics is even less than my mostly imaginary understanding of technology, so I like to compare it to religion because the other ppl on reddit said god is dead. honestly though I would say law is the dark souls of merkle trees

Fair point, but I want to mention: code is law (or vice versa [or something...]), and law has turned to the open source community for code expertise...for religiously political reasons...

Programs, written in code (which is what GitHub is for), are by US (copyright) law defined as "mathematical  equations".

Mathematics (and some physics) is a true pathway to God. It has been deliberately hidden from you by your religious and political elite. It's actually the opposite of politics and religion, which merely sycophantically derive their power from mobs of tricked people.

We've known since Galileo that the Catholic church will murder the truth, and forgive themselves, later... Now they're going to start witch-hunting on GitHub, because everyone had to fight them politically, not mathematically... Shame...

**Comment by CrazedCoding** [comment]
> Speaking seriously, this is one of the problems with the whole crypto scam/fad. People think that anyone who knows some basics of coding somehow are somehow qualified to "disrupt" industries they know nothing about. Humility is all too rare a quality in Silicon Valley.

Because everyone who knows anything about tech is from there, right? Not the places they get their tech from..?

**Comment by jonmarrs** [comment]
> This issue violates community standards.  There isn't a single mention of floppa, and it seems to take seriously the idea that cryptocurrency is somehow valuable.
> 
What makes fiat currency valuable? Why do green pieces of paper with numbers and pictures of dead presidents on them have value? Do you think it is a law of nature that only governments can make money? Why is it that humanity used gold as money for thousands of years? Why have the vast majority of fiat currencies survived for less than 100 years? Does it matter whether the supply of a currency is limited vs. unlimited? Why?
> 
> I do realize I'm going out on a limb, making fun of someone "in tech from SV originally."  After all, there is no finer preparation for creating a new monetary and regulatory system than writing Python scripts and doing QA for websites.
> 
What is adequate preparation for becoming a central banker? Is Jerome Powell, who has a law degree, qualified to be a central banker and steal your time by printing limitless amount of fiat currency out of thin air? Since money is just distilled labor, how much of your time and life's energy can central bankers steal from you via money printing and inflation before you think it is a problem? For example, 2% inflation is like stealing 1 week of your life from you every year. 10% inflation like we have now is like stealing 5 weeks of your life from you every year.  
> 
> Speaking seriously, this is one of the problems with the whole crypto scam/fad.  People think that anyone who knows some basics of coding somehow are somehow qualified to "disrupt" industries they know nothing about.  Humility is all too rare a quality in Silicon Valley.

I think humility is all too rare in central banks. Why should central bankers believe that they should have the godlike power to print trillions of dollars out of thin air and steal billions of years of labor and productivity from people who actually had to work for the same amount of money and are now forced to pay higher prices for gas and food and rent because of inflation?

**Comment by joelsgp** [comment]
so true my brother in bitcoin! satoshi guide us, and may our prayers reach Him as swiftly as the holy transactions! (limited to about 5 per second)

unironially though it's funny how much this guy claims to hate religion since 1. they're the only one in this repository who's brought it up, and 2. bitcoin is a cult (I'll specify that a cult is a kind of *religion*, since the average reading comprehension of a cryptobro must be about 13)

**Comment by CrazedCoding** [comment]
> bitcoin is a cult (I'll specify that a cult is a kind of _religion_, since the average reading comprehension of a cryptobro must be about 13)

Nah. Bitcoin is a mathematical equation.

Law is the religion of politics, and zealots.

https://github.com/responsible-financial-innovation-act22/RFIA-bill/issues/138#issuecomment-1168149528

https://github.com/responsible-financial-innovation-act22/RFIA-bill/issues/138#issuecomment-1168149528

**Comment by joelsgp** [comment]
my goodness me, you're right! Wow. I’ve truly been challenged. Enlightened, even. Who knew the political views of my fellow gamers could be so diverse?
![image](https://user-images.githubusercontent.com/26749912/176195803-dc59d810-452b-4f77-9f48-da0bb7e4d538.png)

**Comment by zathras** [comment]
> > This issue violates community standards.  There isn't a single mention of floppa, and it seems to take seriously the idea that cryptocurrency is somehow valuable.
> 
> What makes fiat currency valuable?

Besides Floppa, you mean?

Being backed by a government with policing powers and the power of taxation (where taxes are levied in said fiat currency) seem to have helped for thousands of years.  Not that it's been a perfect system by any stretch of the imagination, but it has proven to have a certain staying power.

**Comment by jonmarrs** [comment]
> > What makes fiat currency valuable?
> 
> Besides Floppa, you mean?
> 
> Being backed by a government with policing powers and the power of taxation (where taxes are levied in said fiat currency) seem to have helped for thousands of years.  Not that it's been a perfect system by any stretch of the imagination, but it has proven to have a certain staying power.

How did fiat currency work out in the Weimar Republic, Zimbabwe, Venezuela, etc.?
[Most fiat currencies survive less than 35 years](https://ginifoundation.org/kb/fiat-currency-graveyard-a-history-of-monetary-folly/). Fiat currency is the most unsustainable financial system, and always ends in tragedy. Even the [Roman Empire collapsed](https://www.visualcapitalist.com/currency-and-the-collapse-of-the-roman-empire/) partly because it debased its currency. There is a reason gold has survived as money for thousands of years, whereas [virtually every fiat currency has died](https://www.reddit.com/r/Bitcoin/comments/8nwpo9/i_made_an_infographic_there_are_152_fiat/?utm_source=share&utm_medium=ios_app&utm_name=iossmf).

Bitcoin is like gold because it has a limited supply, takes work/effort to create, and isn't controlled by any government.

Centralized control of currencies is why fiat currencies always die, because governments always spend more than they should and debase/print the fiat currency into oblivion.

**Comment by jonmarrs** [comment]
> > so true my brother in bitcoin! satoshi guide us, and may our prayers reach Him as swiftly as the holy transactions! (limited to about 5 per second)
> > unironially though it's funny how much this guy claims to hate religion since 1. they're the only one in this repository who's brought it up, and 2. bitcoin is a cult (I'll specify that a cult is a kind of _religion_, since the average reading comprehension of a cryptobro must be about 13)
> 
> Fiat currency is a cult. His holiness, Jerome Powell, divine conjurer of the fiat, is the cult leader; and fiat worshippers gather to worship him every FOMC meeting and rejoice over his holy proclamations of what the new Fed funds rate will be and the amount of Quantitative Easing / newly printed money that will be deposited into the hands of the wealthy at the expense of the poor. Everyone is forced to pay penance via inflation, for the good of the fiat church. Anyone who dares to question the doctrine of the fiat cult is condemned as a heretic and ostracized like Galileo for speaking truth to power.

The Central Banker is my shepherd; I shall not want.

He maketh me to lie down in green papers: he leadeth me beside the injected liquidity.

He cuteth my interest rates: he leadeth me in the paths of Modern Monetary Theory for his name's sake.

Yea, though I walk through the valley of the shadow of hyperinflation, I will fear no evil: for thou art with me; thy printer and thy Quantitative Easing they comfort me.

Thou preparest a printer before me in the presence of mine debt: thou anointest my Universal Basic Income with fiat; my cost of living runneth over.

Surely fiat and QE shall follow me all the days of my life: and I will dwell in the house of the Federal Reserve for ever.

**Comment by joelsgp** [comment]
![image](https://user-images.githubusercontent.com/26749912/176253054-f4ae92b1-b9f0-455c-88a4-861a88b41fa6.png)

**Comment by joelsgp** [comment]
and for all you catboys, catgirls, and nyanbinary themperors in the audience who are paying attention, GitHub is a web service for hosting Git servers and doing issue tracking, shitposting in crypto threads, kanban, etc. Git is a source control management program. I know my stuff, I read the book, graduated top of my class at harvard business school alma pater 1856

oh, a web service you don't actually seem to use!
![image](https://user-images.githubusercontent.com/26749912/176253523-789c6a58-455b-4096-81b4-fbe11a0274f3.png)

**Comment by CrazedCoding** [comment]
> Yea, though I walk through the valley of the shadow of hyperinflation

😂

**Comment by SuperSpaceMan230** [comment]
![ElQRrLfUcAA1iMt](https://user-images.githubusercontent.com/61877191/177219048-9261a7eb-75ac-42f8-babf-686480d5c25f.jpg)

**Comment by CrazedCoding** [comment]
> ![ElQRrLfUcAA1iMt](https://user-images.githubusercontent.com/61877191/177219048-9261a7eb-75ac-42f8-babf-686480d5c25f.jpg)


![image](https://user-images.githubusercontent.com/18431780/177219492-6e24c634-55fc-4142-8887-c40f64e87c55.png)

This is GitHub. We don't lose nerdy information.

**Comment by CrazedCoding** [comment]
> > ![ElQRrLfUcAA1iMt](https://user-images.githubusercontent.com/61877191/177219048-9261a7eb-75ac-42f8-babf-686480d5c25f.jpg)
> 
> ![image](https://user-images.githubusercontent.com/18431780/177219492-6e24c634-55fc-4142-8887-c40f64e87c55.png)
> 
> This is GitHub. We don't lose nerdy information.

![image](https://user-images.githubusercontent.com/18431780/177219726-8885a12b-76a7-4f03-976d-25f9dc1c36a6.png)

**Comment by CrazedCoding** [comment]
> > > ![ElQRrLfUcAA1iMt](https://user-images.githubusercontent.com/61877191/177219048-9261a7eb-75ac-42f8-babf-686480d5c25f.jpg)
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/18431780/177219492-6e24c634-55fc-4142-8887-c40f64e87c55.png)
> > This is GitHub. We don't lose nerdy information.
> 
> ![image](https://user-images.githubusercontent.com/18431780/177219726-8885a12b-76a7-4f03-976d-25f9dc1c36a6.png)

Also: already an issue: https://github.com/responsible-financial-innovation-act22/RFIA-bill/issues/104#issue-1283651093
